src/MutatorOrchestrator.ts

```diff
@@ -28,24 +29,22 @@ export default class Mutator {
    * @returns {Mutant[]} The generated Mutants.
    */
   mutate(sourceFiles: string[]) {
-    var mutants: Mutant[] = [];
-    var types = _.uniq(_.flatten(_.map(this.mutations, function(mutation: BaseMutation) {
-      return mutation.types;
-    })));
+    let mutants: Mutant[] = [];
 
-    _.forEach(sourceFiles, (sourceFile: string) => {
+    sourceFiles.forEach((sourceFile: string) => {
       try {
-        var fileContent = fileUtils.readFile(sourceFile);
-        var abstractSyntaxTree = parserUtils.parse(fileContent);
-        var nodes = parserUtils.getNodesWithType(abstractSyntaxTree, types);
-        var newMutants = this.findMutants(sourceFile, fileContent, abstractSyntaxTree, nodes);
+        let fileContent = fileUtils.readFile(sourceFile);
```

Comment:
Can't we use an async file io call? Using blocking io can be time consuming when we have hunderds of source files. On the other hand, we can also wait for it to become a problem.